ProteoformSuiteInternal/ExcelReader.cs

```diff
@@ -0,0 +1,126 @@
+﻿using DocumentFormat.OpenXml.Packaging;
```

Comment:
It wasn't showing up in my VS but it was in the proteoformsuiteinternal folder... now deleted!